web/angular-wallet/src/app/setup/account/account.service.ts

```diff
@@ -79,10 +85,22 @@ export class AccountService {
     this.currentAccount.next(account);
   }
 
-  public getAccountTransactions(id: string, firstIndex?: number, lastIndex?: number, numberOfConfirmations?: number, type?: number, subtype?: number): Promise<TransactionList> {
-    const includeTransactions = semver.satisfies(this.selectedNode.version, '>=2.3.1') || undefined;
-    return this.api.account.getAccountTransactions(
-      id, firstIndex, lastIndex, numberOfConfirmations, type, subtype, includeTransactions);
+  public async getAccountTransactions(id: string, firstIndex?: number, lastIndex?: number, numberOfConfirmations?: number, type?: number, subtype?: number): Promise<TransactionList> {
+    try{
+      const includeMultiouts = semver.gte(this.selectedNode.version, constants.multiOutMinVersion, {includePrerelease: true}) || undefined;
```

Comment:
Extended this function to be more robust. Not only relying on version string, but also on server response. If `includeIndirects` won't be accepted, the call is repeated without.